snippets/logging-clusteradmin-access-logs-snip.adoc

```diff
@@ -0,0 +1,36 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+// * modules/cluster-logging-loki-deploy.adoc
+//
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+The error `Parse error: input size too long (XXXX > 5120)` comes when a `cluster-admin` user tries to query application logs for multiple namespaces, where the sum of total characters of all the namespaces in the cluster is more than 5120. To achieve better control over access to logs in LokiStack, the `cluster-admin` user should be a part of the `cluster-admin` group. If the group does not exist, create a new group and add the users.
```

Comment:
I feel like "The error [...] comes" is awkward phrasing. Also, try to rewrite instructions that contain the word "should." From the IBM Style Guide: "Whenever possible, rewrite in a more direct way. Do not use _should_ to refer to an action that must be performed."

I suggest something along these lines:

```adoc
Querying application logs for multiple namespaces as a `cluster-admin` user where the sum total of characters of all of the namespaces in the cluster is greater than 5120 results in the error `Parse error: input size too long (XXXX > 5120)`. For better control over access to logs in LokiStack, make the `cluster-admin` user a member of the `cluster-admin` group. If the `cluster-admin` group does not exist, create it and add the desired users to it.
```

Reply:
Also, do we actually mean to say "make your user a member of the `cluster-admin` group"? That makes more sense to me than adding the user `cluster-admin` to the group `cluster-admin`, as I assume that's automatically how the `cluster-admin` user would work, right?

Reply:
@kowen-rh Makes sense. This is a fix to the error `cluster-admin` users were facing when accessing log in LokiStack, and is fixed by adding them to the group. Your suggestion looks more straight-forward to me so I'm using it. Thanks! :)

Reply:
I have only added the commas to separate the condition. Do let me know if that's okay! 

"Querying application logs for multiple namespaces as a `cluster-admin` user**,** where the sum total of characters of all of the namespaces in the cluster is greater than 5120**,** results in the error `Parse error: input size too long (XXXX > 5120)`."